jerry-core/ecma/base/ecma-errors.c

```diff
@@ -0,0 +1,134 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+  */
+
+#include "ecma-errors.h"
+
+#if JERRY_ERROR_MESSAGES
+
+/**
+ * Error message, if an argument is has an error flag
+ */
+const char * const error_value_msg_p = "Argument cannot be marked as error";
```

Comment:
I would prefix these with `ecma_error_` or something, since we should not take `error_` prefix.